src/lib/activity-store.js

```diff
@@ -4,10 +4,14 @@ import Elasticsearch from '@elastic/elasticsearch';
 class ActivityStore {
 
   constructor(esIndex){
-    this.user = "test";
-    this.pass = "test";
+    this.user = process.env.ELASTICSEARCH_USERNAME || "";
+    this.pass = process.env.ELASTICSEARCH_PASSWORD || "";
     this.esIndex = esIndex;
-    this.client = new Elasticsearch.Client({ node: 'http://localhost:9200' });
+    let client_options = { node: process.env.ELASTICSEARCH_URL || 'http://localhost:9200' };
```

Comment:
clientOptions 
`const` rather than `let` as  we don't want the options modified afterwards. Minor thing - not sure we need to do this as a separate variable as you could pass in on the constructor as previously?

Reply:
I didn't want to add the "auth" bit to the options if there wasn't any (hence let and seperate var) but for all I know elastic is totally cool with that. I'll leave this if that's ok - very minory to fix later anyway